.github/workflows/deploy.yml

```diff
@@ -170,18 +219,44 @@ jobs:
         run: source .venv/bin/activate && python -m pip install --upgrade pip && pip install -r requirements.txt
       - name: build static assets (if frontend)
         if: ${{inputs.assets_required == true}}
-        run: python build.py
+        env:
+          FLASK_ENV: "development"
+        run: source .venv/bin/activate && python build.py
       - name: download previous build
         uses: actions/download-artifact@v2
       - name: Deploy to Gov PaaS
         uses: citizen-of-planet-earth/cf-cli-action@v2
         with:
           cf_api:      ${{secrets.CF_API}}
           cf_org:      ${{secrets.CF_ORG}}
-          cf_space:    ${{secrets.CF_SPACE }}
+          cf_space:    ${{secrets.CF_SPACE}}
           cf_username: ${{secrets.CF_USER}}
           cf_password: ${{secrets.CF_PASSWORD}}
-          command: push ${{inputs.app_name}}-dev --var GITHUB_SHA="${{ github.sha }}"
+          command: push
+                  ${{(inputs.include_manifest) && '-f manifest-dev.yml' || ''}}
+                  ${{inputs.app_name}}-dev
+                  --var GITHUB_SHA="${{ github.sha }}"
+                  ${{(inputs.include_rsa256_base64_public) && format('--var RSA256_PUBLIC_KEY_BASE64={0}', secrets.RSA256_PUBLIC_KEY_BASE64) || ''}}
```

Comment:
Just curious, can we not just check if the secret has been passed in rather than have an input which duplicates this information?